### DISCUSSION TITLE
NGINX reverse proxy

### DISCUSSION THREAD

**Discussion by EffeF** [MainKronos/Sonarr-AnimeDownloader#59] [Q&A]
Vorrei rendere accessibile la pagina web di configurazione `http://mioIP:5000` esternamente all'indirizzo del mio server, ad esempio `https://www.mioIndirizzo/anime`.

Mi servirebbe poter impostare una base url `/anime` in modo che URL locale sia `http://mioIP:5000/anime` analogamente a quanto si può fare ad esempio su Sonarr, è possibile? Se si, come?

**Comment by MainKronos** [comment]
Credo si possa fare con qualche rule di NGINX, ma non l'ho mai fatto quindi non saprei. Prova a guardare la [documentazione](https://docs.nginx.com/).

**Comment by berta24** [comment]
Ma puoi farlo semplicemente aprendo la porta 5000 in UDP sul router e collegandoti con il tuo indirizzo ip pubblico `IP:PORTA`

se il tuo ip pubblico è dinamico ti consiglio di usare un servizio di DNS tipo No-Ip
